README.md

```diff
@@ -43,7 +43,7 @@ To deploy the application:
    application](https://developers.google.com/appengine/docs/python/tools/uploadinganapp) with
 
    ```
-   appcfg.py -A <your-project-id> --oauth2 update .
+   appcfg.py -A <your-project-id> -V v1 update .
```

Comment:
Should we call that out in the text below? eg "if this is your first version deployed, you've got an app! otherwise, you gotta set the version to be default before you can see it, or go to this version-encoded url"